### PR TITLE
Creates tests for panic errors support

### LINKED ISSUE #483: [Solidity Support] Panic Errors
### Problem

The Hedera network supports solidity as one of the smart contracts development languages.
Occasionally items have shown up that surfaced in-equivalence at the language layer.

### Solution

Add examples for panic errors operations

- Add `/contracts/solidity/errors/Panic.sol`
- Cover the tasklist below for 10 panic variations `0x00` to `0x51` . See https://docs.soliditylang.org/en/latest/control-structures.html#panic-via-assert-and-error-via-require 

```[tasklist]
### Panic Error Tasks
- [x] `0x00` Used for generic compiler inserted panics.
- [x] 0x01: If you call assert with an argument that evaluates to false.
- [x] 0x11: If an arithmetic operation results in underflow or overflow outside of an unchecked { ... } block.
- [x] 0x12; If you divide or modulo by zero (e.g. 5 / 0 or 23 % 0).a
- [x] 0x21: If you convert a value that is too big or negative into an enum type.
- [ ] 0x22: If you access a storage byte array that is incorrectly encoded.
- [ ] 0x31: If you call .pop() on an empty array.
- [x] 0x32: If you access an array, bytesN or an array slice at an out-of-bounds or negative index (i.e. x[i] where i >= x.length or i < 0).
- [x] 0x41: If you allocate too much memory or create an array that is too large.
- [x] 0x51: If you call a zero-initialized variable of internal function type.
```

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by konstantinabl**
**Description**:

Creates tests to check panic errors support
**Related issue(s)**:

Fixes #483

### CODE REVIEW

File: test/constants.js [hunk 1]
```diff
@@ -120,7 +120,8 @@ const Contract = {
   AddressContract: 'AddressContract',
   Recipient: 'Recipient',
   Inheritance: 'Inheritance',
-  ControlStructures: 'ControlStructures'
+  ControlStructures: 'ControlStructures',
+  Panic: 'Panic'
```
Comment (Nana-EC) [inline test/constants.js:124]: Add comma so git manages single line updates in future

File: contracts/solidity/errors/Panic.sol [hunk 2]
```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: UNLICENSED
```
Comment (Nana-EC) [inline contracts/solidity/errors/Panic.sol:1]: Apache license

File: test/solidity/errors/panicErrors.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
+const { expect, assert } = require('chai')
```
Comment (Nana-EC) [inline test/solidity/errors/panicErrors.js:1]: File header license is missing

**Review by Nana-EC — COMMENTED** [review]
LG.
A few nits

**Review by Nana-EC — APPROVED** [review]

**Review by georgi-l95 — APPROVED** [review]

**Review by Nana-EC — DISMISSED** [review]
LG, please update PR with descriptions

**Review by georgi-l95 — APPROVED** [review]

**Pull request merged**